examples/lessons/10/app.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-var, max-statements */
+/* eslint-disable array-bracket-spacing, no-multi-spaces */
+/* global document */
+import {
+  GL, AnimationLoop, loadTextures, loadFile, addEvents,
+  resetParameters, setParameters
+} from 'luma.gl';
+
+import {
+  Matrix4
+} from 'math.gl'
+
+import {loadWorldGeometry, World} from './world';
+
+var pitch = 0;
+var pitchRate = 0;
+var yaw = 0;
+var yawRate = 0;
+var xPos = 0;
+var yPos = 0.4;
+var zPos = 0;
+var speed = 0;
+var direction = [0, 0, -1];
+
+const animationLoop = new AnimationLoop({
+  onInitialize: ({canvas, gl}) => {
+    addKeyboardHandler(canvas);
+
+    setParameters(gl, {
+      clearColor: [0, 0, 0, 1],
+      clearDepth: 1,
+      depthTest: true
+    });
+
+    return loadTextures(gl, {
+      urls: ['mud.gif']
+    })
+    .then(textures => {
+		  return loadFile({url: 'world.txt'})
```

Comment:
The indentation looks bad here. We use 2 spaces for indent, no tabs. 

Didn't the linter complain in the pre-commit hook?

Reply:
Done. My editor has tab/space issue, formatted all the text, thanks for reminding. I can't make the linter work on my computer for now.